Update index.md

tmhastings opened the pull request:
- Update index.md file to match the overview file previously edited.
The detailed explanation under "What is Trio?" was NOT altered as this information should be on the home page. The rest of the information is updated to match what is located elsewhere in the documentation.

- links corrected
- orphaned file reference removed

tmhastings committed:
Update index.md

Update index.md file to match the overview file already edited.

tmhastings committed:
Update index.md

Define "ISF"

MikePlante1 commented:
Looks like these links didn't format correctly:
<img width="400" alt="Screenshot 2024-05-09 at 4 42 41 PM" src="https://github.com/nightscout/trio-docs/assets/82073483/ceff6d9f-cc17-42c1-91f6-d77bdf34d035">

Guessing it's because it's markdown inside of html?

marionbarker commented:
The links are not correct, here's an example:

```
https://github.com/nightscout/Trio-docs/blob/Operate-Folder-Updates/docs/EN/settings/configuration/carbratios.md#insulin-carbohydrate-ratio
```

You might mean to use `dev` instead of `Operate-Folder-Updates` for now, with a plan to migrate to `main` soon.
Or perhaps you plan to start with `main` and `dev` at the same version (commit) when you announce for beta testers, in which case, change the links to point to `main.

Either approach works. But please someone needs to check all the links in the entire docs for incorrect links of this type.

MikePlante1 commented:
> The links are not correct, here's an example:
> 
> ```
> https://github.com/nightscout/Trio-docs/blob/Operate-Folder-Updates/docs/EN/settings/configuration/carbratios.md#insulin-carbohydrate-ratio
> ```
> 
> You might mean to use `dev` instead of `Operate-Folder-Updates` for now, with a plan to migrate to `main` soon. Or perhaps you plan to start with `main` and `dev` at the same version (commit) when you announce for beta testers, in which case, change the links to point to `main.
> 
> Either approach works. But please someone needs to check all the links in the entire docs for incorrect links of this type.

Better solution, I think, would be to make the links relative, that way it doesn't link to a specific branch of docs

tmhastings committed:
Update index.md

Update links to internal references

tmhastings committed:
Update index.md

More link corrections

tmhastings committed:
Update index.md

Remove dropdown

tmhastings commented:
I corrected them with the last 3 commits using relative links and removing the dropdowns. I checked that it linked to the correct locations. Please verify it does the same for you (@MikePlante1 or @marionbarker) and submit a review.

MikePlante1 commented:
@tmhastings Links didn't work for me. You should exclude the `/docs/en/` part off. It'll change for different languages anyway, whenever those are added in the future.

I'd also suggest putting the acronym in parenthesis following the full term.

<img width="800" alt="diff" src="https://github.com/nightscout/trio-docs/assets/82073483/68b2751e-5991-4194-8cc2-7f0a35046fc2">

tmhastings committed:
Update index.md

Update CR and ISF references
Correct minor phrasing

tmhastings commented:
> @tmhastings Links didn't work for me. You should exclude the `/docs/en/` part off. It'll change for different languages anyway, whenever those are added in the future.
> 
> I'd also suggest putting the acronym in parenthesis following the full term.
> 
> <img alt="diff" width="800" src="https://private-user-images.githubusercontent.com/82073483/329824866-68b2751e-5991-4194-8cc2-7f0a35046fc2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU2MDU2MzksIm5iZiI6MTcxNTYwNTMzOSwicGF0aCI6Ii84MjA3MzQ4My8zMjk4MjQ4NjYtNjhiMjc1MWUtNTk5MS00MTk0LThjYzItN2YwYTM1MDQ2ZmMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDEzMDIxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThkNzBlMTRkZmYwOTM4ZDM0MzM0NGQ0ZDRjZDc3ZDVhNjJlNDY0ZjJhZjMwNzg3OGNkYjM2MjRjY2FiZWY2MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MMmpMeiQk0BKWy4O0D35cy0F3_1epstJuQ-DT5Y3HAk">
> ```
> - [Carb Ratio (CR)](settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
> - [Insulin Sensitivity Factor (ISF)](settings/configuration/insulinsensitivities.md#insulin-sensitivity-factor)
> - [Basal Rate](settings/configuration/basalprofile.md#basal-profile)
> ```

Done!

tmhastings committed:
Update index.md

Update Work-In-Progress Warning

tmhastings commented:
I also updated the work in progress to reflect the current state of Trio, warn general users against using a beta version, and instruct beta testers on what to do should they locate an error in the documentation. Also added links for quicker referencing.

tmhastings committed:
Update index.md

remove "/docs/EN" from links

marionbarker commented:
I don't know how sphinx works but every other website does not start a relative link with a slash "/".

For example, looking at the raw version of index.md, I see this:
```
Information on [Carb Ratio (CR)](/settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
```

The location of index.md in the folder structure is `docs/EN/index.md`
The location of  carbratios.md in the folder structure is `docs/EN/settings/configuration/carbratios.md`

I would expect the link should be formatted like this: 
```
Information on [Carb Ratio (CR)](settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
```

or 

```
Information on [Carb Ratio (CR)](../settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
```

tmhastings committed:
Update index.md

Correct links

tmhastings commented:
Ok. Hopefully they work now? 😁

marionbarker reviewed (dismissed):
That fixed the links. (I also did a global search and found no more instances of `(/relative-path` in the *.md files.

I don't know if you want to add a commit to remove `Other AIDs <./resources/alternative.md>` because you already deleted the file or open a separate PR.

tmhastings commented:
I can include them in PR #35 as multiple instances of that occurring were indicated by the Loop and Learn team during their review.

tmhastings commented:
> That fixed the links. (I also did a global search and found no more instances of `(/relative-path` in the *.md files.
> 
> I don't know if you want to add a commit to remove `Other AIDs <./resources/alternative.md>` because you already deleted the file or open a separate PR.

I will include it in this PR and tag for your review of the changes.

tmhastings committed:
Update index.md

update toctree to remove orphaned link

MikePlante1 commented on docs/EN/index.md:
```diff
@@ -1,7 +1,8 @@
 :class: caution
 
-**PLEASE NOTE THIS DOCUMENTATION IS STILL BEING WRITTEN BY OUR AWESOME COMMUNITY. UNTIL IT IS FINISHED, PLEASE DO NOT SET UP YOUR TRIO BASED ON THIS DOCUMENTATION ALONE. CONTACT THE COMMUNITY ON DISCORD OR FACEBOOK TO MAKE SURE YOU GET IT RIGHT AND STAY SAFE**
+**GENERAL USERS** This documentation is still under construction during the beta release process. It should not be the main source of information until the main branch of Trio is released. If you have any questions, contact the community on [Discord](https://discord.gg/fCY5svg4) and/or [Facebook](https://www.facebook.com/groups/1351938092206709). Be advised that it is not recommended that general ...
```
Discord link is broken.

tmhastings replied:
Updated with working, Trio Official-generated, set to never expire with unlimited uses link: https://discord.gg/FnwFEFUwXE

tmhastings committed:
Update index.md

Correct discord link

MikePlante1 commented on docs/EN/index.md:
```diff
@@ -1,7 +1,8 @@
 
-**PLEASE NOTE THIS DOCUMENTATION IS STILL BEING WRITTEN BY OUR AWESOME COMMUNITY. UNTIL IT IS FINISHED, PLEASE DO NOT SET UP YOUR TRIO BASED ON THIS DOCUMENTATION ALONE. CONTACT THE COMMUNITY ON DISCORD OR FACEBOOK TO MAKE SURE YOU GET IT RIGHT AND STAY SAFE**
+**GENERAL USERS** This documentation is still under construction during the beta release process. It should not be the main source of information until the main branch of Trio is released. If you have any questions, contact the community on [Discord](https://discord.gg/FnwFEFUwXE) and/or [Facebook](https://www.facebook.com/groups/1351938092206709). Be advised that it is not recommended that gener ...
+**BETA TESTERS** Because this documentation is still under development, if you notice any discrepancies, errors, or edits that need to be made, please submit an [Issue](https://github.com/nightscout/trio-docs/issues/new/choose) on GitHub.
```
If you want `BETA TESTERS` on a new line, add `<br>` to the beginning of line 5

<img width="500" alt="Screenshot 2024-06-19 at 2 31 16 PM" src="https://github.com/nightscout/trio-docs/assets/82073483/b7a3deb8-7259-4b43-a2eb-ecdd85283090">

MikePlante1 committed:
Update index.md

tmhastings commented:
Replaced with PR #48
